xrpl/asyncio/ledger/main.py

```diff
@@ -49,22 +51,32 @@ async def get_latest_open_ledger_sequence(client: Client) -> int:
 
 
 async def get_fee(
-    client: Client, *, max_fee: Optional[float] = 2, fee_type: str = "open"
+    client: Client,
+    *,
+    max_fee: Optional[float] = 2,
+    fee_type: Optional[Literal["open", "minimum"]] = None,
 ) -> str:
     """
-    Query the ledger for the current transaction fee.
+    Query the ledger for the current transaction fee and adjust the fee based on
+    the queue size.
 
     Args:
         client: the network client used to make network calls.
         max_fee: The maximum fee in XRP that the user wants to pay. If load gets too
             high, then the fees will not scale past the maximum fee. If None, there is
             no ceiling for the fee. The default is 2 XRP.
-        fee_type: The type of fee to return. The options are "open" (the load-scaled
+        fee_type: DEPRECATED.
+            The type of fee to return. The options are "open" (the load-scaled
             fee to get into the open ledger) or "minimum" (the minimum transaction
-            fee). The default is "open".
+            fee). The default is `None`.
+
+            Recommended: Do not define any type of return so the
+            fee is calculated more dynamically based on the queue size of the
+            node. It increases the chances for the transaction to succeed.
```

Comment:
Maybe it'd be better to have another option "dynamic" since "open" is the original default? (That way you don't have to use `None` to specify unique behavior, it could just be another mode)